apps/reducers/lib/mix/tasks/etl.stores.ex

```diff
@@ -11,7 +11,7 @@ defmodule Mix.Tasks.Etl.Stores do
     DB.Reducer.State.key(e_ctx, "stats")
   end
 
-  def run(:delete) do
+  def run(["delete"]) do
```

Comment:
Yeah this probably needs work, I made it work..... just not well.

I am not sure where the IO.puts is, but it can be moved somewhere else.